src/tutorial/runtime/runtime-configuration.rst

```diff
@@ -65,7 +65,9 @@ Job Submission
    [runtime]
        [[hello_computehost]]
            script = echo "Hello Compute Host"
-           platform = powerful_computer  # Must be defined in Cylc global config!
+
+           # Must be defined in Cylc global config!
```

Comment:
Easier to get rid of this comment, it's an example, not expected to work.

```suggestion
```

Reply:
I think we should at least refer ahead in the text, with an aside e.g. "Platforms are explained later in ...".